clients/web/src/templates/body/systemConfiguration.pug

```diff
@@ -55,11 +55,16 @@ form.g-settings-form(role="form")
             selected=((settings['core.add_to_group_policy'] ||
           defaults['core.add_to_group_policy']) === "yesmod"))
           | Yes, by group administrators and moderators unless disabled per group
-    .g-collection-create-policy-container.g-settings-form-container
-      label(for="g-core-collection-create-policy") Collection creation policy
-      textarea#g-core-collection-create-policy.form-control
-        = JSON.stringify(settings['core.collection_create_policy'] || defaults['core.collection_create_policy'], null, 4)
-      .g-search-container
+    .g-collection-create-policy-container
+      label 
+        | Collection creation policy
```

Comment:
There's a shortcut for this in pug, which is just

    label Collection creation policy